posts/inside-rust/2020-09-11-stabilizing-intra-doc-links.md

```diff
@@ -0,0 +1,218 @@
+layout: post
+title: "Intra-doc links close to stabilization"
+author: Manish Goregaokar and Joshua Nelson
+team: the rustdoc team <https://www.rust-lang.org/governance/teams/dev-tools#rustdoc>
+---
+
+# Intra-doc links close to stabilization
+
+We're excited to share that intra-doc links are stabilizing soon!
+
+[Intra-doc links] are a feature of `rustdoc` that allow you to link to '[items]' - functions, types, and more - by their name, instead of a hard-coded url. This lets you have accurate links even if your types are [re-exported in a different module or crate][broken-string-links]. Here is a simple example:
```

Comment:
```suggestion
[Intra-doc links] are a feature of `rustdoc` that allow you to link to '[items]' - functions, types, and more - by their name, instead of a hard-coded URL. This lets you have accurate links even if your types are [re-exported in a different module or crate][broken-string-links]. Here is a simple example:
```